🐛Source Google ads: connector doesn't pull data for today (#8225)
airbytehq/airbyte: docs/integrations/sources/google-ads.md

**Note**: Due to constraints from the Google Ads API, the `click_view` stream retrieves data one day at a time and can only retrieve data newer than 90 days ago

**Note**: Due to constraints from the Google Ads API, [metrics](https://developers.google.com/google-ads/api/fields/v8/metrics) cannot be requested for a manager account. Therefore, report streams are only available when pulling data from a non-manager account. 

> **Note**: For incremental streams we sync data up to yesterday using your Google Ads account time zone. Google Ads can filter data only by [date](https://developers.google.com/google-ads/api/fields/v8/ad_group_ad#segments.date) without time. Also, some report cannot load data in real time due to Google Ads [limitations](https://support.google.com/google-ads/answer/2544985?hl=en).

Reviewer: ```suggestion
**Note**: For incremental streams data is synced up to the previous day using your Google Ads account time zone. The reason is that Google Ads can filter data only by [date](https://developers.google.com/google-ads/api/fields/v8/ad_group_ad#segments.date) without time. Also, some report cannot load data in real time due to Google Ads [limitations](https://support.google.com/google-ads/answer/2544985?hl=en).
```

**Note**: Due to constraints from the Google Ads API, the `click_view` stream retrieves data one day at a time and can only retrieve data newer than 90 days ago

> **Note**: Due to constraints from the Google Ads API, [metrics](https://developers.google.com/google-ads/api/fields/v8/metrics) cannot be requested for a manager account. If we try to sync streams with metrics for the manager account, we will get the `REQUESTED_METRICS_FOR_MANAGER` [error](https://developers.google.com/google-ads/api/reference/rpc/v8/QueryErrorEnum.QueryError?hl=en#requested_metrics_for_manager). Metrics present in all report tables and all report streams. All report streams are unavailable for a manager account.

Reviewer: good job adding this here! reducing impl details for user
```suggestion
**Note**: Due to constraints from the Google Ads API, [metrics](https://developers.google.com/google-ads/api/fields/v8/metrics) cannot be requested for a manager account. Therefore, report streams are only available when pulling data from a non-manager account. 
```
